python/pyspark/pandas/missing/indexes.py

```diff
@@ -117,13 +117,10 @@ class MissingPandasLikeDatetimeIndex(MissingPandasLikeIndex):
     to_pydatetime = _unsupported_function("to_pydatetime", cls="DatetimeIndex")
     mean = _unsupported_function("mean", cls="DatetimeIndex")
     std = _unsupported_function("std", cls="DatetimeIndex")
-    map = _unsupported_function("map", cls="DatetimeIndex")
 
 
 class MissingPandasLikeCategoricalIndex(MissingPandasLikeIndex):
-
-    # Functions
-    map = _unsupported_function("map", cls="CategoricalIndex")
+    pass
```

Comment:
I thinks we can remove this and related tests in `../tests/indexes/test_base.py` now.

Reply:
Good idea, I will remove all other MissingPandasLikeCategoricalIndex references as well.